phpunit/class.block-parser-test.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once dirname( dirname( __FILE__ ) ) . '/lib/block-parser.php';
+
+class Block_Parser_Test extends TestCase {
+	private $parser;
+
+	function parse( $input ) {
+		return $this->parser->parse( $input );
+	}
+
+	function setUp() {
+		$this->parser = new Gutenberg_Block_Parser();
+	}
+
+	function test_combinator_succeed() {
+		$this->assertEquals(
+			[ [ 'test', 'bork' ] ],
```

Comment:
Yes, which will need to me merged into the plugin and run against PHP 5.2 along with the rest of the tests.